gst/nnstreamer/nnstreamer.c

```diff
@@ -71,12 +71,44 @@ NNSTREAMER_PLUGIN (tensor_reposrc);
       return FALSE; \
   } while (0);
 
+#if 0
+/**
```

Comment:
I think more explanation is required for this move.

Reply:
In the description of gst_type_find_register()
 * Registers a new typefind function to be used for typefinding. After
 * registering this function will be available for typefinding.
 * This function is typically called during an element's plugin initialization.

Currently nnstreamer does not register type-find function, so I blocked this.

Reply:
I added new PR about type-find function. ( PR #1417 )